### DISCUSSION TITLE
개인사정으로 이번주 북클럽에는 참석 못할 것 같습니다

### DISCUSSION THREAD

**Discussion by wooyaggo86** [ThinkAboutSoftware/AcademicConference#162] [General]
다들 건강하게 지내시나요?
개인적인 사정으로 이번주 모임에는 참여가 힘들 것 같습니다.
일 잘 마무리하고 다다음주에 있는 모임에 참석하겠습니다😢

한국 코로나 감염자수가 심상치 않은데, 다들 건강하세요.
감사합니다

**Comment by jongfeel** [comment]
네 알겠습니다.

사실 무슨 일 있으신가 걱정 되서 물어보려고 했는데, 그래도 사정 있다고 알려 주셔서 감사합니다.

다다음주에 뵐께요
